Libraries/LibJS/Bytecode/Instruction.h

```diff
@@ -88,6 +88,7 @@
     O(JumpGreaterThan)                 \
     O(JumpGreaterThanEquals)           \
     O(JumpIf)                          \
+    O(JumpIfNot)                       \
```

Comment:
Can this be a JumpIf with the targets swapped? :thonk:

Reply:
Oh jeez, how did I not think of that! yeah I think so!

Reply:
> Sidenote from this, but do you think it would be worth having an IsTypeOf instruction to avoid the extra string compare bytecode op for the common pattern I've seen on sites of stuff like
> 
> if (typeof x == 'undefined')
> 
> Doesnt really help on existing benchmarks since its all its all compute heavy, so have not MR'd anything, but figured I would mention it after experimenting. Probably much more worth poking around arrays and that type of thing.
> 
> Fun experiment in any case poking around!

Seems pretty reasonable for cases where we're comparing the result of `typeof` with a constant string literal :)

Reply:
I've applied the change, which is much nicer, but for whatever reason I'm not seeing similar results as before with the overall benchmark even though it was very reproducible for both old and new beforehand. Perhaps it was just fortunate with the way things compiled and was laid out w/ performance lottery?